README.md

```diff
@@ -34,39 +34,39 @@ OpenSSH is installed by default, so please change the mobile/root passwords to p
 ### Installing Android
 Select Project Sandcastle > Setup Project Sandcastle > Start Android.
 When Project Sandcastle has been installed, simply choose "Start Android" to boot back into Android.
 
 **Removing Android:**
-If you wish to remove the Android NAND image and reclaim the space you can login via SSH to your checkra1ned device and mount the final volume and remove the nand file. To do this run `ls /dev/disk0s1s*` and find the last volume. You can verify it's the right volume by running `/System/Library/Filesystems/apfs.fs/apfs.util -p VOLUME_HERE` and if it says Android, that's the correct one. Once you have the volume path you can then run as root (type `su`):
+If you wish to remove the Android NAND image and reclaim the space, you can login via SSH to your Checkra1ned device, and mount the final volume, and remove the NAND file. To do this, run `ls /dev/disk0s1s*` and find the last volume. You can verify it's the right volume by running `/System/Library/Filesystems/apfs.fs/apfs.util -p VOLUME_HERE` and if it says Android, that's the correct one. Once you have the volume path, you can then run these commands as root (type `su`):
 ```
 mkdir -p /tmp/mnt
 mount -t apfs VOLUME_HERE /tmp/mnt
 rm -rf /tmp/mnt/nand
 umount /tmp/mnt
 sync
 ```
-And that will reclaim the space for you.
+and that will reclaim the space for you.
 
 ### Booting Linux
 Select "Project Sandcastle", then "Start Linux".
 
 ## Building
 
 To change the version of checkra1n, edit `CRSOURCE_amd64` and `CRSOURCE_i686`.\
-Execute these commands on a debian-based system.
+Execute these commands on a Debian-based system.
 ```
 git clone https://github.com/raspberryenvoie/odysseyn1x.git
 cd odysseyn1x
 sudo ./build.sh
 ```
 ## Credits
-- Asineth for checkn1x
-- The Checkra1n team for [checkra1n](https://checkra.in)
-- CoolStar for [odysseyra1n](https://github.com/coolstar/Odyssey-bootstrap)
-- [The Procursus team](https://github.com/ProcursusTeam/) for [Procursus](https://github.com/ProcursusTeam/Procursus)
+- Asineth for Checkn1x
+- The Checkra1n team for [Checkra1n](https://checkra.in)
+- CoolStar for [Odysseyra1n](https://github.com/coolstar/Odyssey-bootstrap)
+- [The Procursus Team](https://github.com/ProcursusTeam/) for [Procursus](https://github.com/ProcursusTeam/Procursus)
 - [Corellium](https://github.com/corellium) for [Project Sandcastle](https://projectsandcastle.org)
-- [u/GOGO307](https://www.reddit.com/user/GOGO307/) for the concept of an iso with odysseyra1n
-- [MyCatCondo](https://github.com/MyCatCondo) for suggesting to integrate Project Sandcastle (Android and Linux for the iPhone)
+- [u/GOGO307](https://www.reddit.com/user/GOGO307/) for the concept of an ISO with Odysseyra1n
+- [MyCatCondo](https://github.com/MyCatCondo) for suggesting to integrate Project Sandcastle (Android and Linux for the iPhone and maybe the iPad)
```

Comment:
Another thing, I think that Project Sandcastle doesn't support the iPad: https://projectsandcastle.org/status